presentations/2024-01-25_coffee-and-coding/index.qmd

```diff
@@ -31,11 +31,14 @@ editor:
 
 ::: columns
 ::: {.column width="50%"}
-Tom previously presented about {targets} at a coffee and coding last March and you can revisit his presentation and learn about the reasons why you should use the package to manage your pipeline and see a simple demonstration of how to use the package.  
 
-Matt has presented previously about {targets} and making your workflows (pipelines) reproducible.  
+-   Tom previously presented about {targets} at a coffee and coding last March and you can revisit his presentation and learn about the reasons why you should use the package to manage your pipeline and see a simple demonstration of how to use the package.
+
+-   Matt has presented previously about {targets} and making your workflows (pipelines) reproducible.
+
+-   So..... if you aren't really even sure why your pipeline needs managing as an analyst or whether you actually have one (you do) then links to their presentations are at the end
```

Comment:
For some reason, leaving gaps between bullets means the bullet doesn't line up with the text. Just need to remove the blank lines here and that'll sort it.
![image](https://github.com/The-Strategy-Unit/data_science/assets/18232097/de7d4f02-5fda-43f4-be4a-1c3be5712db3)

Reply:
Yeah, I realised this but couldnt figure out what was going on!